peer_manager.go

```diff
@@ -130,13 +156,18 @@ func (pm *peerManager) peersAddrs() []ma.Multiaddr {
 // 	return pm.addFromMultiaddrs(cfg.ClusterPeers)
 // }
 
-func (pm *peerManager) addFromMultiaddrs(addrs []ma.Multiaddr) error {
+// this resets peers!
+func (pm *peerManager) addFromMultiaddrs(addrs []ma.Multiaddr, save bool) error {
```

Comment:
This name is misleading and we might regret it later.  Could we change to something else?  Possibly `setFromMultiaddrs`?

Reply:
fixed